modules/light-clients/08-wasm/client/cli/tx.go

```diff
@@ -53,3 +59,65 @@ func newStoreCodeCmd() *cobra.Command {
 
 	return cmd
 }
+
+// newStoreCodeGovCmd returns the command to send a proposal to store new wasm bytecode.
+func newStoreCodeGovCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "store-code [path/to/wasm-file]",
```

Comment:
Until we remove the other CLI, maybe we can rename the command to `legacy-store-code`, so that we can still use it when doing manual testing and we don't need to way for gov to approve. It just makes the manual testing a bit easier...

Reply:
yea yea good idea, renamed old to add the legacy prefix